### ISSUE TITLE
AWS_S3 plugin.. Changing the CDN_Url in the plugin parameters completely breaks everything.

### ISSUE THREAD

**Issue by SireWilliamson** [WWBN/AVideo#7798]
**Describe the bug**
I am using a digital ocean space cdn to store my video files.
I recently attempted to move to a different space, which has a different CDN_Link
Changing the CDN_Link causes ALL videos to cease to function.

I tried manually changing the object data it in the plugins table, and it doesn't actually do anything, so I'm not sure what the object data is even used for.

**To Reproduce**
1. Go to 'plugins panel'
2. Click on 'edit parameters' for the AWS_S3 plugin
3. Scroll down to 'CDN_Link'
4. Change the url to something new.
EG CDN_Link: https://old.cdn.com
     CDN_Link: https://new.cdn.com

**Expected behavior**
All video URLS update from https://old.cdn.com/filename.mp4
to https://new.cdn.com/filename.mp4

**Observed Behavior**
All video URLS update from https://old.cdn.com/filename.mp4
to https://new.cdn.com     (No file names.)

**Problem can be refersed by simply putting the old Link back in.
I do not understand the behaviour, and since I can't see the plugin code, I can't find out what is going wrong or how to fix it.

**Comment by DanielnetoDotCom** [comment]
The correct parameter should be CDN_Link, there is no CDN_URL

![image](https://user-images.githubusercontent.com/18009293/225163906-0c18946e-1b41-488f-b9c0-73bc55d0ba28.png)

**Comment by SireWilliamson** [comment]
Yes, sorry my mistake, that is the field I meant. But the bug is still there, i just called it cdn_url in stead of cdn_link. If you change the cdn_link, all videos cease to work.

**Comment by DanielnetoDotCom** [comment]
can you please screenshot your parameters screen? (do not forget to hide the keys)

I tested it here and works fine. make sure you have the latest AWS_S3 and also the latest AVideo git pull

**Comment by SireWilliamson** [comment]
I am running the latest of both AVideo and AWS_S3 plugin.
But since you said it was working for you, I thought, it must be something in the input.
I discovered that if you put in a CDN_Link without a trailing "/" everything breaks, but if you include the "/" it works.
EG "https://new.cdn.com" - All video links break.
"https://new.cdn.com/" - Links are changed to match new CDN location.

This is confusing since the "endpoint:" doesn't need the trailing "/".

I would suggest adding a filter, or making a hover note for users. Just a suggestion.
Thanks for the quick response though.

**Comment by DanielnetoDotCom** [comment]
in the new version I will make sure I will correct that, thanks
